### ISSUE TITLE
Can your internet speed determine the bots you can use ?

### ISSUE THREAD

**Issue by ALBABOY** [davidmann4/ogar-feeder-bot#492]
Hello World,i just wanted to ask you guys does the speed determine the bots you use ?
I gathered proxys from some Websites and put them all together and i got about 100-150 working bots.
My friend asked me for the bots and  i gave him the files .He told me that he only gets about 30-40 bots and they are lagy af,his internet speed is about 5 mb down and 2mb up mine is 100mb down 10mb up.Hope someone volunteers :smile:

**Comment by hassanila** [comment]
Yes it definetely depends on your bandwidth..  more bandwidth = more bots....  Why not use a vps?  There you can often get up to 300 bots with no problem even with your slow internet.

**Comment by DeadWarrior58** [comment]
@hassanila97 can you recommend a cheap ,but good, vps?

**Comment by SanjaySasimandir** [comment]
@ALBABOY yes internet speed also determines the number of bots you can use.
If you have low internet speed and if you try to get more bots your game will get laggy making it impossible to play. 
You can use a VPS if you like. Also try to get good proxies that has less ping and fast connection speeds. You can use http://proxybag.blogspot.com/ or http://bulksocks5.blogspot.com/ or http://www.vipsocks24.net/, these are really good sites for finding proxies.
Hope this helps XD

**Comment by easyrider81** [comment]
I use a vps and can get up to 500 bots or so but if you want to be able to control them and not have delayed response when pressing split etc or just not lag the server to fuck then less is better. 200 works Good for me.

**Comment by hassanila** [comment]
Cheap and reliable vps is OVH

**Comment by hassanila97** [comment]
Well I can get 500 bots no lag on 2 servers at the same time..   250/server

**Comment by easyrider81** [comment]
I'm using OVH aswell actually. not tried on 2 servers though.
